### PR TITLE
OM-535 | SEO optimization.

### PR DESCRIPTION
**Pull request by tuovinensanttu**
Most of the texts are just placeholders for now & all the images and twitter related stuff will be added later. I added `Disallow: /` to robots.txt, which should block search engine access for now.

### CODE REVIEW

**Comment by badrange** [comment]
This looks good - in Finnish

Did you consider other languages or is that out of scope for this issue?

**Comment by tuovinensanttu** [comment]
> This looks good - in Finnish
> 
> Did you consider other languages or is that out of scope for this issue?

Oh yeah, that totally skipped my mind! I'll add those as well!

**Comment by Jokinen** [comment]
Interesting comment from @badrange 

Are social media parsers generally able to go through JS code? This template is static and for non-js crawlers will only ever this content, right 🤔

But maybe it's whatever--Google can crawl the description even if it is set from `react-helmet`.

**Comment by badrange** [comment]
I haven't found a way to make social media sharing work, so I've just put Finnish into index.html since most of our users are Finnish speakers. 

But Google SERP works nicely with react-helmet!

**Comment by badrange** [comment]
So if the scope of this issue is sharing, it is now done, if the scope is looking like "something" on google search result pages, react-helmet will help

**Comment by tuovinensanttu** [comment]
I think adding react-helmet is in the scope of this issue and needs to be done sooner or later. I'll continue working on this issue.

File: src/common/layout/PageLayout.tsx [hunk 1]
```diff
@@ -12,19 +14,28 @@ type Props = {
...
-          : styles.adultBackground
-      )}
-    >
-      <Header />
-      <HostingBox className={styles.hostingBox}>{props.children}</HostingBox>
-      <Footer />
-    </div>
+    <HelmetProvider>
+      <Helmet>
+        <html lang={i18n.languages[0]} />
```
Comment (Jokinen) [inline src/common/layout/PageLayout.tsx:22]: Good job on including this 👏

File: src/common/layout/PageLayout.tsx [hunk 2]
```diff
@@ -12,19 +14,28 @@ type Props = {
...
+        <title>{t('appName')}</title>
+        <meta name="description" content={t('login.helpText')} />
```
Comment (Jokinen) [inline src/common/layout/PageLayout.tsx:24]: I expected to see open graph tags here as well 🤔

Comment (tuovinensanttu) [reply src/common/layout/PageLayout.tsx:24]: Good point, added tags to this file as well.

**Pull request merged**